vmap/toy/vmap_toy.go

```
@@ -35,7 +35,7 @@ func main() {
 		glog.Fatalf("Failed create MapStorage: %v", err)
 	}
 
-	hasher := merkle.NewMapHasher(merkle.NewRFC6962TreeHasher())
+	hasher, _ := merkle.Factory("RFC6962-SHA256")
```

Don't ignore the error here